landing_page/app/views/microposts/index.html.erb

```diff
@@ -32,6 +32,11 @@
                 <%= micropost.updated_at %>
                 <br></br>
                 <%= link_to 'Show', micropost%>
+                <% if logged_in? %>
+                    <li class="edit_post">
```

Comment:
If you're using `<li>` tags here maybe the 'Show' link should also be in an `<li>` tag for consistency? Obviously this is a style thing and doesn't really impact the functionality, so it's up to you if you change it.